rr_viz/src/rr_viz/widgets/mission_dashboard/mapping/slam_supervisor.py

```diff
@@ -52,6 +54,9 @@ def __init__(self, parent=None):
             self.slam_sup_name+"/save_map", String)
         self.default_map_name = rospy.get_param("~default_map_name", "")
 
+        # Setup rostopiclabel
+        self.modeLabel.setup(self.slam_sup_name+"/mode")
```

Comment:
This label isn't actually used anywhere. I pressume we should have some code in the runMapping and runLocalization functions to set this label to either Mapping or Localization. I quickly tried it with the line ' self.modeLabel.setText("Mapping")' after line 98 in this file and it appears in rr_viz when in mapping.

Reply:
But adding them doesn't write to the label at first launch, only when you click a button to swtich

Reply:
this might be just back end implemented by @JuliusSustarevas

Reply:
check the rostopiclabel class. The class is  a (label + subscriber to a string topic) whatever is on the topic, thats whats being shown in the label. I think this label was created to address the story that it is not clear when you are in mapping and when in localisation.  The topic is being published by slam_supervisor  (not rrviz slam_supervisor, but the one @msadowski  wrote. )

Reply:
@JuliusSustarevas @pepeRossRobotics  When i tested it, nothing appeared saying whether you were in mapping or localization. I've fixed it on my current branch so can push those changes onto this branch for review it you want? Or wait till i finish the work i'm doing